src/Microsoft.DocAsCode.Dfm/DfmCodeRenderer.cs

```diff
@@ -39,6 +40,30 @@ public virtual StringBuffer Render(IMarkdownRenderer renderer, DfmFencesToken to
             }
         }
 
+        public virtual StringBuffer RenderFencesFromCodeContent(string queryStringAndFragment, string codeContent)
+        {
+            if (string.IsNullOrEmpty(codeContent))
+            {
+                throw new ArgumentNullException(nameof(codeContent));
+            }
+
+            if (string.IsNullOrEmpty(queryStringAndFragment))
+            {
+                throw new ArgumentNullException(nameof(queryStringAndFragment));
+            }
+
+            var pathQueryOption =
+                !string.IsNullOrEmpty(queryStringAndFragment)
+                    ? DfmFencesRule.ParsePathQueryString(queryStringAndFragment.Remove(1), queryStringAndFragment.Substring(1))
```

Comment:
I misunderstand it, so "ab" will be OK?

---
In reply to: [134933830](https://github.com/dotnet/docfx/pull/1996#discussion_r134933830) [](ancestors = 134933830)